### DISCUSSION TITLE
Subfolder List on Files page

### DISCUSSION THREAD

**Discussion by raflymln** [directus/directus#15573] [Feature Requests]
Hi, I think it'd be great if we implemented subfolder list interface on files page like how Google Drive do, what i mean is:
![image](https://user-images.githubusercontent.com/19299378/190168185-77c5a269-78d3-4fa6-8028-c50d07617639.png)

The implementation i thought for Directus should looks like this:
![image](https://user-images.githubusercontent.com/19299378/190169173-59a60d37-4177-4eb9-89ba-0bee4b5af481.png)

I mean it's already great to have a list on the left menu, but what if you had _a lot of subfolder, under a subfolder, under a subfolder, under a subfolder, and so on_... some people just want everything to look neat right? 😅, it would looks kinda confusing if you only put the list of subfolder on the sidebar.

Would like to see this feature implemented in the future actually 🙏

**Comment by azrikahar** [comment]
This feature is actually discussed in https://github.com/directus/directus/discussions/4999#discussioncomment-601368 and #3464 😄

**Reply by raflymln** [reply]
Ah, okay, didn't know about that, thanks for the info!

**Comment by rijkvanzanten** [comment]
Duplicate of #4999
